src/journeyQuery.ts

```diff
@@ -0,0 +1,48 @@
+export const getJourneyQuery = () => {
+  return `
+  query planTrip($fromPlace: String, $toPlace: String, $numTripPatterns: Int, $dateTime: DateTime, $arriveBy: Boolean, $pageCursor: String){
```

Comment:
Excuse the nitpick 🙈 But since it won't be fixed by eslint:

```suggestion
query planTrip($fromPlace: String, $toPlace: String, $numTripPatterns: Int, $dateTime: DateTime, $arriveBy: Boolean, $pageCursor: String) {
```